doc/nrf/device_guides/working_with_nrf/nrf70/developing/soc_combo.csv

```diff
@@ -0,0 +1,4 @@
+Stack components,Wi-Fi SSID scanning with cellular backhaul,Wi-Fi application for end-device-to-cloud products,Wi-Fi + BLE application for end-device-to-cloud products,Matter over Wi-Fi products,Matter over Thread products,Dual-stack capable Matter devices (not gateways)
```

Comment:
For both the tables, please use Bluetooth LE instead of BLE:
Wi-Fi + Bluetooth LE